lms/djangoapps/instructor/views/data_access_constants.py

```diff
@@ -49,7 +49,7 @@ class Inclusion:
                          'F': Inclusion.FILTER}
 
 INCLUDE_SECTION_PATTERN = re.compile('chapter|sequential')
-INCLUDE_PROBLEM_PATTERN = re.compile('problem')
+INCLUDE_PROBLEM_PATTERN = re.compile('problem|openassessment|submit-and-compare|freetextresponse|xblockmufi|qualtricssurvey|inline-dropdown|poll|image-coding')
```

Comment:
I wonder if it would be easier to read/maintain if we broke it up, something like:

```python
INCLUDE_PROBLEMS = [
    'problem',
    'openassessment',
    'submit-and-compare',
    # etc...
]
INCLUDE_PROBLEM_PATTERN = re.compile('|'.join(INCLUDE_PROBLEMS))
```